solidity/contracts/AugurConstantProductMarket.sol

```diff
@@ -26,26 +25,26 @@ contract AugurConstantProduct is ERC20 {
 
 	function addLiquidity(uint256 sharesToBuy) external {
 		// TODO: gas golf this function a bit, though it doesn't really matter
-		uint256 poolConstantBefore = sqrt(poolConstant());
+		uint256 poolConstantBefore = sqrt(getPoolConstant());
```

Comment:
I think I would prefer to rename the local variables that conflicted with this rather than the public variable.  We calculate internally as a performance optimization, so even if we referenced the wrong thing it would still give the right answer.  Can we silence the warning somehow (easily)?